frontend/__tests__/components/chat/chat-interface.test.tsx

```diff
@@ -56,6 +56,7 @@ describe("Empty state", () => {
           content: "Hello",
           imageUrls: [],
           timestamp: new Date().toISOString(),
+          secondaryId: crypto.randomUUID(),
```

Comment:
Instead of doing this, can we just show the message when it comes back to us on the websocket?

Reply:
I have updated the PR with this change - we now have a `pending` flag (On the client only), and when a message from the server is received, any `pending` message on the client is removed.